spam/01_spam_tutorial.py

```diff
@@ -863,7 +863,7 @@ def plot_probabilities_histogram(Y):
 # We'll use Keras, a popular high-level API for building models in TensorFlow, to build a simple logistic regression classifier.
 # We compile it with a `categorical_crossentropy` loss so that it can handle probabilistic labels instead of integer labels.
 # Using a _noise-aware loss_ &mdash; one that uses probabilistic labels &mdash; for our discriminative model lets
-# us take full advantage of the label model's learning procedure (see our [NeurIPS 2016 paper](https://arxiv.org/abs/1605.07723)).
+# us take full advantage of the label model's learning procedure (see our [NeurIPS 2016 paper](https://arxiv.org/abs/1605.07723).
```

Comment:
Shouldn't we have a closing bracket for the (see our.. ?

Reply:
ohhhhhh whoops...my regexing failed...